OmiseSwiftTests/ChargesOperationFixtureTests.swift

```diff
@@ -1097,6 +1097,54 @@ class ChargesOperationFixtureTests: FixtureTestCase {
         XCTAssertEqual(items[5].value, "1")
     }
     
+    func testEncodingCreatePromptPayChargeParams() throws {
+        let params = ChargeParams(value: Value(amount: 10_000_00, currency: .thb),
+                                  sourceType: .promptPay,
+                                  chargeDescription: "Test",
+                                  metadata: ["customer_id": "123"])
+        
+        let encoder = URLQueryItemEncoder()
+        encoder.arrayIndexEncodingStrategy = .emptySquareBrackets
+        
+        let items = try encoder.encode(params)
+        
+        XCTAssertEqual(items.count, 5)
+        XCTAssertEqual(items[0].name, "amount")
+        XCTAssertEqual(items[0].value, "1000000")
+        XCTAssertEqual(items[1].name, "currency")
+        XCTAssertEqual(items[1].value, "THB")
+        XCTAssertEqual(items[2].name, "source[type]")
+        XCTAssertEqual(items[2].value, "promptpay")
+        XCTAssertEqual(items[3].name, "description")
+        XCTAssertEqual(items[3].value, "Test")
+        XCTAssertEqual(items[4].name, "metadata[customer_id]")
+        XCTAssertEqual(items[4].value, "123")
+    }
+    
+    func testEncodingCreatePayNowChargeParams() throws {
+        let params = ChargeParams(value: Value(amount: 10_000_00, currency: .thb),
```

Comment:
PayNow is specifically a Singapore payment method, so using "THB" instead of "SGD" as currency is a bit misleading.  In fact, our API won't allow using "THB" as currency for a PayNow charge.

Reply:
True, good point. I changed it to "SGD".